custom_typings/spdy.d.ts

```diff
@@ -0,0 +1,148 @@
+// copied from node's "http2" module, which should have same API as spdy
```

Comment:
typings for pem: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11666

typings for node-spdy in progress...